sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala

```diff
@@ -333,6 +333,72 @@ abstract class OrcQueryTest extends OrcTest {
     }
   }
 
+  test("dpp") {
+    // 4 rows, cells of column 1 of row 2 and row 4 are null
+    val fact = (0 to 99).map { i =>
+      (i, i + 1, (i + 2).toByte, (i + 3).toShort, (i * 20) % 100, (i + 1).toString)
+    }
+
+    val dim = (0 to 9).map { i =>
+      (i, i + 1, (i + 2).toByte, (i + 3).toShort, (i * 10), (i + 1).toString)
+    }
+
+    withOrcTable(fact, "fact", true, Seq.apply("_1", "_2", "_3")) {
+      withOrcTable(dim, "dim") {
+        val df = sql(
+          """
+            |SELECT f._1, f._2, f._3, f._4 FROM fact f
+            |JOIN dim d
+            |ON (f._2 = d._2)
+            |WHERE d._5 > 80
+            """.stripMargin)
+        val explainDF = df.queryExecution.explainString(ExplainMode
+          .fromString("extended"))
```

Comment:
It seems that your IDE has a different setting. Apache Spark uses 100 .

https://github.com/apache/spark/blob/43f650eaa43f0586e0066118a1f53f8b1f20f88f/scalastyle-config.xml#L78

Could you double-check your code and adjust like the following in order to follow the community coding guidelines?
```scala
-        val explainDF = df.queryExecution.explainString(ExplainMode
-          .fromString("extended"))
+        val explainDF = df.queryExecution.explainString(ExplainMode.fromString("extended"))
```